### ISSUE TITLE
State of the lib/repo. Is this still actively maintained?

### ISSUE THREAD

**Issue by Ohjurot** [cnjinhao/nana#676]
Dear @cnjinhao,
Dear @qPCR4vir,
Dear nana contributors and community,

I'm using nana (1.7.4) for quite a while now. 
Unfortunately, I can't see any activity from the repository owner since mid 2022. In fact, I don't see any online activity from Jinhao in the known channels at all. 
I'm wondering how the state of this lib is. I see that development of 1.8 has "stopped" with the lack of activity from Jinhao in 2022 (A year ago). The last official release was in 2020 (which does not even build without modifications...). I have seen that you Ariel have done many commits in the past and that you also have an active ongoing PR, so maybe you can help me as well. 

**Now the big question: Can we either,**
- **get feedback from Jinhao what the current state of the lib is.**
- **or get feedback from somebody that knows what the current problem is.**

I really hope that all issues can be resolved and that nana can move forward. I see many opportunities for this lib and would really like to contribute some of my ideas. However, that's only possible if this lib will still be actively maintained!

Thanks in Advance
Ludwig F.

You can also contact me here, in case you don't want to give a public comment: ludwig.fuechsl@moxibyte.com

**Comment by cnjinhao** [comment]
Yes, it's still actively maintained. Now, the virtual keyboard is supported for touchscreen input, this feature has been merged into develop-1.8

![virtual keyboard](https://github.com/cnjinhao/nana/assets/1246418/429372d6-df41-44b9-bf73-d746af49a605)
